Client/Frontend/Theme/ThemeManager.swift

```diff
@@ -52,19 +57,21 @@ final class DefaultThemeManager: ThemeManager, Notifiable {
 
     // MARK: - Init
 
-    init(userDefaults: UserDefaults = UserDefaults.standard,
+    init(userDefaults: UserDefaults = UserDefaults(suiteName: AppInfo.sharedContainerIdentifier)!,
```

Comment:
Why not the standard anymore?

Reply:
Also, what about the `!`? Is this something we're good with in prod?
Also also, for testing, I added `UserDefaultsInterface`. I don't know if that's helpful here or not, but it exists, so I'm just letting people know. :)

Reply:
This is the user defaults suite that is used by Prefs. I would like to have all of the state logic self contained in the theme and not rely on the profile object and I would also like to avoid needing to any sort of migration if I can help it, so this means I need to use the same user defaults suite as Prefs. 
You are probably right about the force unwrap. NSUserDefaultsPrefs also force unwraps this so in theory the app will have already crashed by this point if it can't create this but it's a bad pattern unless the only good option is to crash the app. I will provide the standard user defaults as a fallback.